internal/security/secretstore/constants.go

```diff
@@ -19,10 +19,8 @@
 package secretstore
 
 const (
-	// KVSecretsEngineMountPoint is the name of the mount point base for Vault's key-value secrets engine
-	KVSecretsEngineMountPoint = "secret"
-	// ConsulSecretEngineMountPoint is the name of the mount point base for Vault's Consul secrets engine
-	ConsulSecretEngineMountPoint = "consul"
+	// this is a feature key to indicate whether to enable Registry Consul's ACL or not
+	RegistryACLFeatureFlag = "ENABLE_REGISTRY_ACL"
```

Comment:
Remove. This feature doesn't need to be in secretstore setup.  Secretstore-setup doesn't enable the Consul ACL feature.  We can create the token for configuring the consul secrets provider regardless of whether or not we enable Consul ACLS.

Reply:
I disagree, if no ACL enable why need to create this token at all?

Reply:
resolved, also use this same flag to not install Cosnul's secret engine